src/map/mob.c

```diff
@@ -4024,16 +4307,26 @@ static void mob_read_db_drops_sub(struct mob_db *entry, struct config_setting_t
 			i++;
 			continue;
 		}
-		if (mob->get_const(drop, &i32) && i32 >= 0) {
-			value = i32;
+
+		struct optdrop_group *drop_option = NULL;
+		if (config_setting_is_number(drop)) {
+			// Setting is a number, item doesn't contain options
+			if (mob->get_const(drop, &i32) && i32 >= 0) {
```

Comment:
What happens if i32 is a negative value? perhaps throwing an error and put a default value.

Reply:
same as in 4245, being negative keeps `value` to 0 and throws a warning on line 4322.